dist/index.js

```diff
@@ -36493,7 +36493,7 @@ const supportedSchemas = new Set(['data:', 'http:', 'https:']);
  * @param   {*} [options_] - Fetch options
  * @return  {Promise<import('./response').default>}
  */
-async function fetch(url, options_) {
+async function src_fetch(url, options_) {
```

Comment:
why snake_case?

Reply:
ah wait, this is a dist .js file, is it on purpose that it is not in the gitignore?